README.md

```
@@ -285,12 +328,12 @@ https://fulfillment-internal-api.osac.svc.cluster.local:8000
 The same secret is shared by all service accounts and users.
 
-The `--ca-file` flag should point to a file containing the trusted CA certificates. In the default
-installation, the CA bundle is stored in the `ca-bundle` ConfigMap created by _trust-manager_. You
-can extract it with:
+The `--ca-file` flag should point to a file containing the trusted CA certificates. If you used the
+`IT_CA_CRT` environment variable, you can point directly to that file. Otherwise, the CA bundle is
+stored in the `ca-bundle` ConfigMap created by _trust-manager_. You can extract it with:
 
 ```bash
-$ kubectl get configmap ca-bundle -n osac -o jsonpath='{.data.bundle\.pem}' > bundle.pem
+$ kubectl get configmap ca-bundle -n osac -o json | jq -r '.data["bundle.pem"]' > ca.crt
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)